bundles/org.openhab.binding.venstarthermostat/pom.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+  <modelVersion>4.0.0</modelVersion>
+    <build>
```

Comment:
This can be removed

Reply:
Hi @digitaldan- 

Thanks so much for the tips. I'll incorporate those and push an update. Stay tuned!